DESIGN.md

```diff
@@ -0,0 +1,63 @@
+# ODD
+
+## Introduction
+
+ODD stands for Open Distributed Data. We want our data accessible from anywhere and distributed, meaning that it can live partially or fully in multiple places. The ODD SDK does this through the [WNFS protocol](https://github.com/wnfs-wg).
```

Comment:
expand a little bit: "The ODD SDK is a toolkit, currently implemented in Typescript, that helps developers build systems that store data in a secure, reliable, and decentralized way. The approach ODD uses relies on open protocols, including WNFS and UCAN"

/ or something like that, just a bit more context